ibis/backends/pandas/execution/window.py

```diff
@@ -202,18 +202,20 @@ def trim_with_timecontext(data, timecontext: Optional[TimeContext]):
     # noop if timecontext is None
     if not timecontext:
         return data
+
     # reset multiindex and turn series into a dateframe
     df = data.reset_index()
-    # if Series dosen't contain a name, reset_index will assign
-    # '0' as the column name for the column of value
-    name = data.name if data.name else 0
 
     # Filter the data, here we preserve the time index so that when user is
     # computing a single column, the computation and the relevant time
     # indexes are returned.
     time_col = get_time_col()
-    if time_col not in df or not is_datetime64_dtype(df[time_col]):
+    if time_col not in df or not is_datetime64_any_dtype(df[time_col]):
```

Comment:
Do we have test for this

Reply:
Removed this, not hitting by any user level APIs